fast/stages/00-bootstrap/README.md

```diff
@@ -358,10 +358,20 @@ federated_identity_providers = {
   github-sample = {
     attribute_condition = "attribute.repository_owner==\"my-github-org\""
     issuer              = "github"
+    issuer_uri          = null
```

Comment:
You also need to add the fields to the `federated_identity_providers` variable type, or it will be discarded